rurusetto/users/templates/users/profile.html

```diff
@@ -56,7 +56,12 @@ <h1>created rulesets <span class="badge rounded-pill bg-rurusetto round-font-bol
         <div class="row text-center gy-4">
             {% for detail in created_ruleset %}
             <div class="col-lg-4" data-aos="fade-up" data-aos-duration="800" data-aos-once="true">
+
+            {% if user.config.theme == 'light' %}
             <img src="{{ detail.0.icon.url }}" alt="{{ detail.0.name }}" width="140px" height="140px" class="rounded-circle">
+            {% else %}
+            <img src="{{ detail.0.dark_icon.url }}" alt="{{ detail.0.name }}" width="140px" height="140px" class="rounded-circle">
+            {% endif %}
```

Comment:
```suggestion

            {% if user.config.theme == 'light' %}
            <img src="{{ detail.0.light_icon.url }}" alt="{{ detail.0.name }}" width="140px" height="140px" class="rounded-circle">
            {% else %}
            <img src="{{ detail.0.icon.url }}" alt="{{ detail.0.name }}" width="140px" height="140px" class="rounded-circle">
            {% endif %}
```

Because the website's default theme is dark so it need to change.